Emby.Server.Implementations/SyncPlay/SyncPlayController.cs

```diff
@@ -194,26 +194,24 @@ private GroupUpdate<T> NewSyncPlayGroupUpdate<T>(GroupUpdateType type, T data)
         }
 
         /// <inheritdoc />
-        public void InitGroup(SessionInfo session, CancellationToken cancellationToken)
+        public void CreateGroup(SessionInfo session, CancellationToken cancellationToken)
         {
             _group.AddSession(session);
             _syncPlayManager.AddSessionToGroup(session, this);
 
             _group.PlayingItem = session.FullNowPlayingItem;
-            _group.IsPaused = true;
+            _group.IsPaused = session.PlayState.IsPaused;
             _group.PositionTicks = session.PlayState.PositionTicks ?? 0;
             _group.LastActivity = DateTime.UtcNow;
 
             var updateSession = NewSyncPlayGroupUpdate(GroupUpdateType.GroupJoined, DateToUTCString(DateTime.UtcNow));
             SendGroupUpdate(session, BroadcastType.CurrentSession, updateSession, cancellationToken);
-            var pauseCommand = NewSyncPlayCommand(SendCommandType.Pause);
-            SendCommand(session, BroadcastType.CurrentSession, pauseCommand, cancellationToken);
         }
 
         /// <inheritdoc />
         public void SessionJoin(SessionInfo session, JoinGroupRequest request, CancellationToken cancellationToken)
         {
-            if (session.NowPlayingItem?.Id == _group.PlayingItem.Id && request.PlayingItemId == _group.PlayingItem.Id)
+            if (session.NowPlayingItem?.Id == _group.PlayingItem.Id)
```

Comment:
Why remove this check?

Reply:
We use the session in every one of our major clients, which will properly report the currently playing song used in the first check. That makes the second check redundant for all those clients. In addition, this entire conditional is only used as a convenience when someone is already playing the group item so it doesn't have to reload, which is only relevant for video streams. Not only that, it would only be useful if the client was at the exact same position as the group, since otherwise it would have to load new chunks anyways. Even if all of that was true and it was a perfect storm of events (client doesn't implement the session, playing the same exact item as the group, includes the parameter in its request, at the exact same seek position as the group) it would still be mostly useless since it will be *entirely* handled by the client. When the conditional triggers, it merely sends a *suggestion* to change the currently playing item, which could then be handled arbitrarily, including ignoring it completely.